version.sh

```diff
@@ -0,0 +1,4 @@
+PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]')
+sed 's/VERSION \= \"[0-9\.]*\"/VERSION = \"'${PACKAGE_VERSION}'\"/g' package.bzl > package.bzl.new
```

Comment:
use `sed -i` to do the replacement inline so you don't need tmp file?

Reply:
thought about that and even tried it but it looks like `-i` is a standard sed option between distros and it varies in how its used... safer to just do the `mv`